src/main/java/org/generationcp/middleware/manager/InventoryDataManagerImpl.java

```diff
@@ -241,27 +206,6 @@ public Integer apply(@Nullable final TransactionReportRow input) {
 		return transactionDetailsForLot;
 	}
 
-	@Override
-	public List<Germplasm> getAvailableBalanceForGermplasms(final List<Germplasm> germplasms) {
-		final List<Integer> gids = new ArrayList<>();
-
-		for(final Germplasm germplasm : germplasms) {
-			gids.add(germplasm.getGid());
-			germplasm.setInventoryInfo(new GermplasmInventory(germplasm.getGid()));
-		}
-
-		final Map<Integer, Object[]> availableBalanceCountAndTotalLotsCount =
-			this.daoFactory.getLotDao().getAvailableBalanceCountAndTotalLotsCount(gids);
-
-		for(final Germplasm germplasm : germplasms) {
-			final Object[] availableBalanceValues = availableBalanceCountAndTotalLotsCount.get(germplasm.getGid());
-			this.getListInventoryBuilder().setAvailableBalanceAndScale(germplasm.getInventoryInfo(), availableBalanceValues);
-		}
-
-		this.getListInventoryBuilder().setAvailableBalanceScaleForGermplasm(germplasms);
-		return germplasms;
-	}
-
 	@Override
 	public Map<Integer, String> retrieveStockIds(final List<Integer> gids){
```

Comment:
this one can be removed too @lik-83 (commented by slack)

Reply:
Done!